src/ResourceManager/ApiManagement/Commands.SMAPI.Test/ScenarioTests/ApiManagementTests.ps1

```diff
@@ -304,6 +304,43 @@ Param($resourceGroupName, $serviceName)
 
 <#
 .SYNOPSIS
+Tests API import from Wsdl and create a Rest Api
+#>
+function Api-ImportWsdlToCreateSoapToRestApi
+{
+Param($resourceGroupName, $serviceName)
+
+    $context = New-AzureRmApiManagementContext -ResourceGroupName $resourceGroupName -ServiceName $serviceName
+
+	$wsdlUrl = "http://www.webservicex.net/stockquote.asmx?WSDL"
```

Comment:
Can we guarantee that this URL will always be around?  Static urls generally create a maintenance problem.  Consider making this a parameter to the test (passed in through an envionment variable or the liek), unless this is a permanent fixture.